### ISSUE TITLE
Make names of markers changeable

### ISSUE THREAD

**Issue by cameris** [octgn/OCTGN#1156]
In The Spoils cards or its effects sometimes require special handling. I give the following cards as examples to explain why I need to be able to change the names of markers:

Incriminating Photograph (Item)
When this card enters play, name a non-resource card.
The named card cannot be played.

Pant Swipe (Tactic)
Remove the top 4 cards of your opponent's deck from the game facedown.
For the rest of the game, you may play the removed cards as though they were in your hand even if you cannot meet their threshold.

I Drink Your Milkshake (Tactic)
You may search your opponent's deck for a card. If you do, remove it from the game facedown. For the rest of the game, if your opponent plays a card, you may reveal the removed card if it has the same name. If you do, you may play it for free.

As you may see, we cant add cards removed from game by Pant Swipe and "Milkshake" to the general "Out of Game"-Pile. While the named card from Incriminating Photograph would have to be remembered as long as it stays on table.
Currently this is realized by custom markers as you can see in the image:
![current](https://f.cloud.github.com/assets/5940815/1728480/a20378cc-62af-11e3-9f7c-f9078251aee2.jpg)

for the "out of game"-cards, "oog" is prepented to the actual tactic name.
this is used for the autoplacments of cards with a function:
```python
def has_oog_marker(card): # {{{
	for marker in card.markers:
		if marker[0].lower()[:3] == 'oog':
			return True
	return False # }}}
```

To make them visually more distinguishable for players I would like to use other Icons, from the "All Markers" section instead:
![markers](https://f.cloud.github.com/assets/5940815/1728542/b0661f4a-62b0-11e3-8698-8a5c14e8bc95.jpg)
For example if i use the "Can't play" marker and hover over the marker it would always display "Can't play (1x)" even if I used:
```python
card.markers[("Plunging Shriever", "DONT_PLAY_GUID")] = 1
```

What I would like to have:
![want](https://f.cloud.github.com/assets/5940815/1728603/e5efdb96-62b1-11e3-9625-762df6e879ca.jpg)

Which could be achieved by the following function:
```python
def addmarker(cards, x = 0, y = 0):
	mute()
	marker, quantity = askMarker()
	if quantity == 0: return
	name, guid = marker
	if guid == "DONT_PLAY_GUID" or guid == "OOG_GUID":
		newname = askString() # see  Issue #1155
		if newname:
			name = newname
	for c in cards:
		c.markers[(name, guid)] += quantity
		notify("{} adds {} {} markers to {}.".format(me, quantity, name, c))
```
After those markers are placed on the cards, the name doesn't need to be changed anymore.
I hope this illustrates what I was trying to explain in the IRC.

**Comment by brine** [comment]
I think the problem is, when OCTGN tries to identify the marker it looks at the GUID first, then matches it to an existing marker defined in the set defs.  Then it just uses the name in there, instead of the name that python gives it.

So if we force the name that python gives the marker, then maybe it will solve itself?

**Comment by brine** [comment]
I believe this was indirectly addressed by the fix for #446.  Please reopen if its not the case.
